### PR TITLE
Hyperlinked the novu logo.

### LINKED ISSUE #1848: 🚀 Feature: add link to the logo
### 🔖 Feature description

image

### 🎤 Why is this feature needed ?

,

### ✌️ How do you aim to achieve this?

,

### 🔄️ Additional Information

_No response_

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by Abubakrce19**
Now the user can go to the official website of novu with a single click.
In Readme.md file

- **Why was this change needed?** (You can also link to an open issue here)
Resolved #1848

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,12 @@
 </br>
 
 <div align="center">
+  <a href="https://novu.co">
```
Comment (davidsoderberg) [inline README.md:17]: What do you think of doing target _blank?

**Review by davidsoderberg — APPROVED** [review]
Added a small comment 😄

File: README.md (hunk 1 above)
Comment (Abubakrce19) [reply README.md]: > What do you think of doing target _blank?

You are absolutely right.
Done

**Comment by Abubakrce19** [comment]
@davidsoderberg 
Changes done 
Please have a look now!

**Comment by davidsoderberg** [comment]
@Abubakrce19 I tested it now in preview mode and I can not get the link to work because of target _blank maybe Github disables it?

File: README.md [hunk 2]
```diff
@@ -14,10 +14,12 @@
 </br>
 
 <div align="center">
+  <a href="https://novu.co target="_blank">
```
Comment (davidsoderberg) [inline README.md:17]: A " is missing here...

**Comment by Abubakrce19** [comment]
@davidsoderberg 
Thank you for the merge.
Am i eligible for the swags now?
Do this pr counted?

**Comment by davidsoderberg** [comment]
This PR is counted :)

**Pull request merged**